packages/site/src/data/graphql/libraries.ts

```diff
@@ -1,26 +1,490 @@
+		href: "https://www.apollographql.com/docs/apollo-server/",
+		description:
+			"Apollo Server is an open-source, spec-compliant GraphQL server that's compatible with any GraphQL client, including Apollo Client. It's the best way to build a production-ready, self-documenting GraphQL API that can use data from any source.",
+		image: "https://github.com/apollographql.png",
+		tags: ["Servers"],
+	},
+	{
+		name: "Helix",
+		author: "Contra",
+		gitHubRepo: "https://github.com/contra/graphql-helix",
+		npmPackage: "graphql-helix",
+		href: "https://www.graphql-helix.com/",
+		description:
+			"Build your own extensible and framework-agnostic GraphQL Server",
+		image:
+			"https://raw.githubusercontent.com/contra/graphql-helix/main/logo.svg",
+		tags: ["Servers"],
+	},
+	{
+		name: "Express-GraphQL",
+		author: "Meta",
+		gitHubRepo: "https://github.com/graphql/express-graphql",
+		npmPackage: "express-graphql",
+		href: "https://github.com/graphql/express-graphql",
+		description:
+			"Create a GraphQL HTTP server with any HTTP web framework that supports connect styled middleware, including Connect itself, Express and Restify.",
+		image: "https://github.com/graphql.png",
+		tags: ["Servers"],
+	},
+	{
+		name: "GraphQL-Go",
+		author: "graph-gophers",
+		gitHubRepo: "https://github.com/graph-gophers/graphql-go",
+		npmPackage: "",
+		href: "https://github.com/graph-gophers/graphql-go",
+		description:
+			"The goal of this project is to provide full support of the GraphQL draft specification with a set of idiomatic, easy to use Go packages.",
+		image: "https://github.com/graph-gophers.png",
+		tags: ["Servers"],
+	},
+	{
+		name: "Yoga",
+		author: "Dotan Simha",
+		gitHubRepo: "https://github.com/dotansimha/graphql-yoga",
+		npmPackage: "graphql-yoga@three",
+		href: "https://www.the-guild.dev/graphql/yoga-server",
+		description:
+			"Fully-featured GraphQL Server with focus on easy setup, performance & great developer experience",
+		image: "",
+		tags: ["Servers"],
+	},
+	{
+		name: "GraphQL .NET",
+		author: "",
+		gitHubRepo: "https://github.com/graphql-dotnet/graphql-dotnet",
+		npmPackage: "",
+		href: "https://graphql-dotnet.github.io/",
+		description: "This is an implementation of Facebook's GraphQL in .NET.",
+		image: "https://github.com/graphql-dotnet.png",
+		tags: ["Servers"],
+	},
+	{
+		name: "GraphQL Hooks",
+		author: "NearForm",
+		gitHubRepo: "https://github.com/nearform/graphql-hooks",
+		npmPackage: "",
+		href: "https://github.com/nearform/graphql-hooks",
+		description: "Minimal hooks-first GraphQL client.",
+		image: "graphql-hooks",
+		tags: ["Clients"],
+	},
+	{
+		name: "GraphQL-request",
+		author: "Prisma Labs",
+		gitHubRepo: "https://github.com/prisma-labs/graphql-request",
+		npmPackage: "graphql-request",
+		href: "https://github.com/prisma-labs/graphql-request",
+		description:
+			"Minimal GraphQL client supporting Node and browsers for scripts or simple apps",
+		image: "",
```

Comment:
Please add an image here.

Reply:
There is a lot of Libraries without images, I'm gonna add the image of the github user instead but this may need another PR in the Framework.dev core to set a default image